File: src/fizzbuzz.m
#import "fizzbuzz.h"

@implementation fizzbuzz
for (int i = 1; i <= 100; i++)
{
	if(i % 15 == 0 && i % 5 == 0) {
		NSLog("fizzbuzz");
	}
	else if(i % 3 == 0) {
		NSLog("buzz");
	}
	else if(i % 5 == 0) {
		NSLog("fizz");
	}
	else {
		NSLog(@"%d", i);
	}
	NSLog(@"%d", i);
}
@end

File: src/fizzbuzz.R
# install [MAC]: brew install R
# to run: R --no-save < fizzbuzz.r
i <- 1

while(i <= 100){
	if(i %% 5 == 0 && i %% 3 == 0) {
		cat(noquote('fizzbuzz'), "\n")
		} else if(i %% 3 == 0) {
			cat(noquote('fizz'), "\n")
			} else if(i %% 5 == 0) {
				cat(noquote('buzz'), "\n")
			}
			else {
				cat(i, "\n")
			}
	i <- i+1
}
